carbons/carbons.go

```diff
@@ -61,3 +61,65 @@ func DisableIQ(ctx context.Context, s *xmpp.Session, iq stanza.IQ) error {
 	)), &v)
 	return err
 }
+
+
+//DisableCarbon is an xmlstream.Transformer that inserts <private xmlns='urn:xmpp:carbons:2'/>
+//into an individual message, which in turn disables carbons for that message. If a message contains
+//<private xmlns='urn:xmpp:carbons:2'/> already then we will not add it. Other we will add it to everything
+//for now and sort it out later
```

Comment:
A few minor nits about the help text:

- there should be a space between each `//` and the start of the line (this keeps the comments readable when scanning the source code)
- text in comments should be wrapped to 80 characters (same as above; this just keeps things readable)
- Please start a newline after each sentence (this makes diffs easier to read by breaking up lines around where changes are likely to happen)
- I think the name of this function (in the comment and function body) should be `DisableCarbons` (with an "s")

Sorry, I know these comments sound nitpicky but they help us keep the code readable and easy to work on for new contributors in the future!

Reply:
Not a problem at all, I appreciate you giving me an opportunity to gain experience working on open source projects and with go.

Reply:
Looking at this with fresh eyes this morning `DisableCarbons` won't be a good name either because the full name as used by other packages will be `carbons.DisableCarbons` which is redundant. What do you think about naming it `Exclude`, as in `carbons.Exclude` instead?

Reply:
That sounds good. I will rename the function to Exclude